q2_diversity/tests/test_alpha.py

```diff
@@ -27,9 +27,10 @@ def test_alpha(self):
         t = biom.Table(np.array([[0, 1, 3], [1, 1, 2]]),
                        ['O1', 'O2'],
                        ['S1', 'S2', 'S3'])
-        actual = alpha(table=t, metric='observed_otus')
+        actual = alpha(table=t, metric='feature_counts')
```

Comment:
Can you add a couple quick tests for alpha and beta diversity that asserts the error message is rewritten?